### PR TITLE
Adds specific version picker to install script.

### PR DESCRIPTION
**Pull request by weymanf**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -13,21 +13,20 @@ fail() {
 }
 
 usage() {
-  log "$0 [--head]"
+  log "$0 [--head][--version]"
```
Comment (AlexDenisov) [inline install.sh]: Space between `][`?

Comment (weymanf) [inline install.sh]: Well that should be just print list of commands that are possible. I don't really know how to show that in command lines.

**Comment by akitchen** [comment]
Hey, thanks @weymanf !

Your second commit has August's name on it, not you - is that right?

Also @AlexDenisov 's suggestion, just a space between ```]``` and ```[``` for better output.

**Comment by weymanf** [comment]
No. This was pretty much made by me and Vinson. I forgot to git pair.

**Comment by jeffh** [comment]
Thanks @weymanf.

I've merged the PR with the modification requested by @akitchen and @AlexDenisov.

**Pull request merged**